### PR TITLE
Fix issue #36: Anagram check

### LINKED ISSUE #36: Anagram check
Provide snippet that checks if word is anagram of another word

https://en.wikipedia.org/wiki/Anagram

### PR DESCRIPTION
**Pull request by PalAditya**
Fixes #36 
Note: Apparently var in arrays is not allowed, so I used int declaration there, although I couldn't find these behavior in the docs 🤔

### CODE REVIEW

**Comment by PalAditya** [comment]
@iluwatar please review this.

File: README.md [hunk 1]
```diff
@@ -392,3 +393,24 @@ For new snippets the general steps are
...
 [⬆ back to top](#table-of-contents)
+
+### Anagram Check
+
+```java
+public boolean isAnagram(String s1, String s2) {
+	var l1 = s1.length();
+	var l2 = s2.length();
+	int [] arr1 = new int[256];
+	int [] arr2 = new int[256];
```
Comment (iluwatar) [inline README.md]: var arr1 = new int[256];

Comment (iluwatar) [reply README.md]: $ jshell
|  Welcome to JShell -- Version 11.0.4
|  For an introduction type: /help intro

jshell> var arr1 = new int[256];
arr1 ==> int[256] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ... , 0, 0, 0, 0, 0, 0, 0, 0 }

Comment (PalAditya) [reply README.md]: Thanks for the information. Fixed 😄

**Comment by iluwatar** [comment]
Excellent @PalAditya, thank you very much for this snippet :relaxed:

**Pull request merged**